pom.xml

```diff
@@ -142,6 +142,26 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <version>3.0.0</version>
+        <executions>
+          <execution>
+            <id>add-source</id>
+            <phase>generate-sources</phase>
```

Comment:
We should be adding the generated sources after actually generating them, so let's try something after `generate-sources`, like `process-sources` maybe?

Reply:
ok